examples/script/concolic.py

```diff
@@ -95,174 +97,149 @@ def eq(a, b):
         return False
 
     return True
-    
-
-def eqls(a, b):
-	# a b and 2 iterables of branch constraints
-    for aa, bb in zip(a, b):
-        if not eq(aa, bb):
-            return False
-    return True
 
+def perm(lst, func):
```

Comment:
yep. if you think of the index of the loop below as the elements of `lst` to flip, it skips the first bitmask (all zeroes, i.e. no flips).

Reply:
super clever! can we add just a few words in the docstring to document the cleverness, since i get the feeling i will wut pretty hard at this code in 6 months